features/GroupInvites/joinGroup/JoinGroupNavigation.tsx

```diff
@@ -21,20 +22,23 @@ export const JoinGroupScreenConfig = {
 
 export function JoinGroupNavigation() {
   const colorScheme = useColorScheme();
+
   return (
-    <NativeStack.Screen
-      name="GroupInvite"
-      component={JoinGroupScreen}
-      options={({ route }) => ({
-        headerShadowVisible: false,
-        headerTitle: "",
-        headerTintColor:
-          Platform.OS === "android"
-            ? textSecondaryColor(colorScheme)
-            : textPrimaryColor(colorScheme),
-        animation: navigationAnimation,
-        headerTitleStyle: headerTitleStyle(colorScheme),
-      })}
-    />
+    <NativeStack.Group screenOptions={stackGroupScreenOptions(colorScheme)}>
```

Comment:
Not sure why we need a Group here 🤔 

But okay for now since we'll need to do a refactor on the navigation anyway